requirements_test.txt

```diff
@@ -0,0 +1,8 @@
+coverage==4.4.1
+mock>=1.0.1
+flake8>=2.1.0
+tox>=1.7.0
+codecov>=2.0.0
+
+
+# Additional test requirements go here
```

Comment:
for consistency with other openwisp modules, could you please rename to `requirements-test.txt`?

Reply:
sure